images/02-autoformat/auto-format.sh

```diff
@@ -23,10 +23,10 @@ for dev in ${DEVS[@]}; do
             # do not auto-format if the disk does not begin with 1MB filled with 00
             continue
         fi
-    
-        mkfs.ext4 -L RANCHER_STATE ${dev}
-    
+
+
         if [ -e "/userdata.tar" ]; then
+            mkfs.ext4 -L B2D_STATE ${dev}
```

Comment:
@joshwget why did you choose a different label (B2D_STATE) when preserving the contents of the volume? I am missing some context and did not see it referenced in the issues.